spark/src/main/scala/geotrellis/spark/reproject/TileRDDReproject.scala

```diff
@@ -72,7 +73,12 @@ object TileRDDReproject {
     val passthroughGridExtent = ReprojectRasterExtent(sourceDataGridExtent, metadata.crs, destCrs)
     val targetDataExtent = passthroughGridExtent.extent
 
-    val sourcePartitioner = bufferedTiles.partitioner
+    val targetPartitioner: Option[Partitioner] =
```

Comment:
`Option` has handy [`orElse`](http://www.scala-lang.org/api/2.11.8/index.html#scala.Option@orElse[B>:A](alternative:=>Option[B]):Option[B]) that enables `partitioner.orElse(bufferedTiles.partitioner)`

Reply:
Didn't know about that method, but it looks pretty handy.